contracts/BSCValidatorSet.sol

```diff
@@ -67,6 +79,15 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     uint256 incoming;
   }
 
+  // BEP-127 Temporary Maintenance
+  struct ValidatorExtra {
+    uint256 enterMaintenanceHeight;     // the block number at which the validator enters Maintenance
```

Comment:
`the block number at which the validator enters Maintenance` ==> `the block number from where the validator enters Maintenance`

Reply:
fixed